### PR TITLE
feat: add test usecase to query interface

### PR DESCRIPTION
**Pull request by heiliuchao**
Signed-off-by: heiliuchao <heiliuchao110@163.com>

### CODE REVIEW

File: opengemini_client/query_test.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        cfg = models.Config(address=[models.Address(host='127.0.0.1', port=8086)],
+                            auth_config=models.AuthConfig(auth_type=models.AuthType(0)),
+                            batch_config=models.BatchConfig(batch_size=10, batch_interval=10),
+                            timeout=datetime.timedelta(seconds=30), connection_timeout=datetime.timedelta(seconds=10),
+                            gzip_enabled=False, tls_enabled=False
+                            )
+        cli = client_impl.OpenGeminiDBClient(cfg)
+        qr, _ = cli.query(query=models.Query('', 'create database test_openGeminiDB_1', ''))
+        print(qr)
+        self.assertEqual(qr.results, {'results': [{'statement_id': 0}]})
```
Comment (zhiheng123) [inline opengemini_client/query_test.py]: should be :
```
self.assertEqual(qr.results['results'], [{'statement_id': 0}])
```

File: opengemini_client/query_test.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        qr, _ = cli.query(query=models.Query('', 'show databases', ''))
+        series_str = qr.results['results'][0]
```
Comment (zhiheng123) [inline opengemini_client/query_test.py]: should be:
```
        result = qr.results['results'][0]
        self.assertTrue(['test_openGeminiDB_1'] in result["series"][0]['values'])
```

File: opengemini_client/client_impl.py [hunk 3]
```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
             return None
         return Error(f"ping openGeminiDB status is {resp.status_code}")
 
-    def query(self, query: Query) -> tuple:
-        return ()
+    def query(self, query: Query):
+
+        if query.command.lower().startswith("select") and "into" not in query.command.lower():
```
Comment (zhiheng123) [inline opengemini_client/client_impl.py]: This logic is incorrect. For example, show database will be judged as a POST type

File: opengemini_client/client_impl.py [hunk 4]
```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
...
+            method = 'GET'
+        else:
+            method = 'POST'
+
+        server_url = self.get_server_url()
+        url_path = UrlConst.QUERY
```
Comment (zhiheng123) [inline opengemini_client/client_impl.py]: The variable 'url_path' is unnecessary and can be inlined.

File: opengemini_client/client_impl.py [hunk 5]
```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
...
+
+        params = {'db': query.database, 'q': query.command, 'rp': query.retention_policy}
+
+        resp, error = self.request(method=method, server_url=server_url, url_path=url_path, params=params)
+
+        if error is not None:
```
Comment (zhiheng123) [inline opengemini_client/client_impl.py]: This is unnecessary and can be deleted.

File: opengemini_client/client_impl.py [hunk 6]
```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
             return None
         return Error(f"ping openGeminiDB status is {resp.status_code}")
 
-    def query(self, query: Query) -> tuple:
-        return ()
+    def query(self, query: Query):
+
```
Comment (zhiheng123) [inline opengemini_client/client_impl.py]: Redundant line

File: opengemini_client/client_impl.py [hunk 7]
```diff
@@ -96,19 +97,21 @@ def update_headers(self, method, url_path, headers=None) -> dict:
...
                 body = compressed.getvalue()
+        if params is None:
+            params = {}
 
-        req = requests.Request(method, full_url, data=body, headers=headers)
+        req = requests.Request(method, full_url, data=body, headers=headers, params=params)
         prepared = req.prepare()
         try:
-            resp = self.session.send(prepared)
+            resp = self.session.send(prepared, verify=False)
```
Comment (hezhangjian) [inline opengemini_client/client_impl.py]:
```suggestion
            resp = self.session.send(prepared, verify=False)
```
why add `verify=False` ?

File: opengemini_client/client_impl.py (hunk 5 above)
Comment (heiliuchao) [reply opengemini_client/client_impl.py]: line 148-149 should be deleted

File: opengemini_client/client_impl.py (hunk 6 above)
Comment (heiliuchao) [reply opengemini_client/client_impl.py]: Seems to be left over from conflict resolution。local repo is correct。

File: opengemini_client/client_impl.py (hunk 3 above)
Comment (heiliuchao) [reply opengemini_client/client_impl.py]: ok

File: opengemini_client/client_impl.py (hunk 4 above)
Comment (heiliuchao) [reply opengemini_client/client_impl.py]: I has been inlined

**Pull request merged**